Plugins/Flow.Launcher.Plugin.BrowserBookmark/Commands/Bookmarks.cs

```diff
@@ -6,19 +6,19 @@ namespace Flow.Launcher.Plugin.BrowserBookmark.Commands
 {
     internal static class Bookmarks
     {
-        internal static bool MatchProgram(Bookmark bookmark, string queryString)
+        internal static MatchResult MatchProgram(Bookmark bookmark, string queryString)
         {
-            if (StringMatcher.FuzzySearch(queryString, bookmark.Name).IsSearchPrecisionScoreMet()) return true;
-            if (StringMatcher.FuzzySearch(queryString, bookmark.PinyinName).IsSearchPrecisionScoreMet()) return true;
-            if (StringMatcher.FuzzySearch(queryString, bookmark.Url).IsSearchPrecisionScoreMet()) return true;
-
-            return false;
+            var match = StringMatcher.FuzzySearch(queryString, bookmark.Name);
+            if (match.IsSearchPrecisionScoreMet())
+                return match;
+            else
```

Comment:
Since you are doing a 'return' here, you dont need an 'else'

Reply:
I will remove it. I keep the else because I think it will make the code looks more logical.

Reply:
oh ok, sorry, i guess it is a matter of personal preference. Yeah up to you then, i am not too fussed.